src/h264packetizationhandler.cpp

```diff
@@ -0,0 +1,164 @@
+/*
+ * libdatachannel client example
+ * Copyright (c) 2020 Filip Klembara (in2core)
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "h264packetizationhandler.hpp"
+
+#if RTC_ENABLE_MEDIA
+
+using namespace rtc;
```

Comment:
I don't really like `using namespace` to define methods as it kind of defeats the purpose of namespaces (of course it's fine for actually using methods). I prefer defining them in the namespace with `namespace rtc { ... }`.

Reply:
Fixed